services/export/shared/helpers/file.js

```diff
@@ -113,56 +52,47 @@ const list = (dirPath, count = 100, page = 0) => new Promise((resolve, reject) =
 	);
 });
 
-const purge = (dirPath, days) => new Promise((resolve, reject) => {
+const purge = async (dirPath, days) => {
 	if (days === undefined) throw new ValidationException('days cannot be undefined');
-	fs.readdir(dirPath, async (err, files) => {
-		if (err) {
-			logger.error(err);
-			return reject(err);
-		}
-		await BluebirdPromise.map(
-			files,
-			async (file) => {
-				const stat = await getFileStats(path.join(dirPath, file));
-				const currentTime = new Date().getTime();
-				const expirationTime = new Date(stat.ctime).getTime() + getDaysInMilliseconds(days);
-				try {
-					if (currentTime > expirationTime) await remove(path.join(dirPath, file));
-				} catch (error) {
-					logger.error(err);
-				}
-			},
-			{ concurrency: files.length },
-		);
-		return resolve();
-	});
-});
-
-const exists = async filePath => !!(await fs.promises.stat(filePath).catch(() => null));
-
-const isFile = async (filePath) => {
-	const isExists = await exists(filePath);
-
-	if (isExists) {
-		try {
-			const stats = await fs.promises.lstat(filePath);
-			return stats.isFile();
-		} catch (error) {
-			return false;
-		}
+  
+	try {
+	  // Get the list of files in the directory using getFiles method
+	  const files = await getFiles(dirPath, { withFileTypes: true });
+  
+	  // Calculate the expiration time for each file and remove if expired
+	  await BluebirdPromise.map(
+		files,
+		async (filePath) => {
+		  const stat = await stats(filePath);
+		  const currentTime = new Date().getTime();
+		  const expirationTime = new Date(stat.ctime).getTime() + getDaysInMilliseconds(days);
+  
+		  try {
+			if (currentTime > expirationTime) await rm(filePath);
+		  } catch (error) {
+			logger.error(error);
+		  }
+		},
+		{ concurrency: files.length },
+	  );
+  
+	  return;
+	} catch (err) {
+	  logger.error(err);
+	  throw err;
 	}
+  };
 
-	return false;
-};
+const init = async (params) => mkdir(params.dirPath);
 
 module.exports = {
 	init,
 	write,
 	read,
-	remove,
+	remove: rm,
 	list,
 	purge,
-	exists,
+	existsFile,
```

Comment:
Must be `exists`!?

Reply:
We have multiple functionalities. One to check if a file exists and other to check if file and a folder exists.
https://github.com/LiskHQ/lisk-service/blob/117bc58e6358f693b6a7e48e80849e69c0bef5af/framework/src/fs.js#L99